.claude/skills/git-convention/SKILL.md

```
@@ -0,0 +1,110 @@
+
+## 제목 포맷 3종 — 미묘하게 다르다
+
+**이슈에는 번호가 없고, PR에는 번호가 붙고, 커밋만 type이 소문자**다.
+
+| 대상 | 포맷 | 예 |
+|------|------|-----|
+| 이슈 | `[{Type}] {한국어 설명}` | `[Fix] JWT Token 재발급이 안되는 오류 수정` |
+| PR | `#{이슈번호} [{Type}] {한국어 설명}` | `#260 [Fix] JWT Token 재발급이 안되는 오류 수정` |
+| 커밋 | `#{이슈번호} [{type}] {한국어 설명}` | `#260 [feat] 알 수 없는 리프레쉬 토큰 오류메시지 추가` |
+
+- **PR 제목 = 이슈 제목 앞에 `#{이슈번호} `를 붙인 것.** 번호를 빠뜨리거나(`[Feat] ...`) 뒤에 붙이지(`... (#355)`) 않는다.
```

markdownlint-cli2 (0.23.0)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)